test/Microsoft.VisualStudio.Composition.Tests/StaticMemberExportsTests.cs

```diff
@@ -194,5 +194,99 @@ public class ImportManyWithMetadataPart
             [MefV1.ImportMany("Property")]
             public List<Lazy<string, IDictionary<string, object>>> ImportingMember { get; set; } = null!;
         }
+
+        // Test class to verify static members don't cause instantiation
+        public class ClassWithStaticMemberExports
+        {
+            private static bool constructorCalled = false;
+
+            public static bool ConstructorCalled
+            {
+                get { return constructorCalled; }
+                set { constructorCalled = value; }
+            }
+
+            public ClassWithStaticMemberExports()
+            {
+                constructorCalled = true;
+            }
+
+            [MefV1.Export("StaticField")]
+            public static string StaticField = "StaticFieldValue";
+
+            [MefV1.Export("StaticProperty")]
+            public static string StaticProperty => "StaticPropertyValue";
+
+            [MefV1.Export("StaticMethod")]
+            public static string StaticMethod() => "StaticMethodValue";
+        }
+
+        // Test class with mixed static and instance exports
+        public class ClassWithMixedExports
+        {
+            private static bool constructorCalled = false;
+
+            public static bool ConstructorCalled
+            {
+                get { return constructorCalled; }
+                set { constructorCalled = value; }
+            }
+
+            public ClassWithMixedExports()
+            {
+                constructorCalled = true;
+            }
+
+            [MefV1.Export("StaticMixed")]
+            public static string StaticExport = "StaticValue";
+
+            [MefV1.Export("InstanceMixed")]
+            public string InstanceExport => "InstanceValue";
+        }
+
+        [MefFact(CompositionEngines.V3EmulatingV1, typeof(ClassWithStaticMemberExports))]
```

Comment:
We should probably start by running against all 3 MEF engines (and add MEFv2 attributes to your test classes). Then let the test result instruct us as to the behavior of MEFv1 and MEFv2. Then we can exclude those engines if they behave differently.